README.md

```diff
@@ -2,7 +2,6 @@
 
 Django content management as it should be.
 
-[![Build Status](https://travis-ci.org/5monkeys/djedi-cms.svg?branch=master)](https://travis-ci.org/5monkeys/djedi-cms)
```

Comment:
Think we should use something like this:
`[![Build Status](https://github.com/5monkeys/djedi-cms/workflows/CI/badge.svg)](https://github.com/5monkeys/djedi-cms/actions)`

Same links in appropriate places in long_description in `setup.py` as well